### PR TITLE
Fix: Flaky Duration Test

### PR DESCRIPTION
**Pull request by dmoree**
# Description

This PR address #765 where a specific integration test was randomly exposing a floating point error in `Duration`. The error has been addressed and the test values have been converted to definite values.

# Issue

- #765

### CODE REVIEW

**Review by angrykoala — CHANGES_REQUESTED** [review]
The changes in this PR look like a step in the right direction and seem to fix #765
Just a comment and one change requested in the modified test

File: packages/graphql/tests/integration/types/duration.int.test.ts [hunk 1]
```diff
@@ -52,12 +51,10 @@ describe("Duration", () => {
...
-            const years = faker.random.number({ min: -6, max: 6 });
-            const months = faker.random.number({ min: -10, max: 10 });
-            const days = faker.random.number({ min: -50, max: 50 });
-            const minutes = faker.random.float();
+            const hours = 2;
+            const minutes = 30;
 
-            const duration = `P${years}Y${months}M${days}DT${minutes}M`;
+            const duration = `PT${hours}H${minutes}M`;
             const parsedDuration = parseDuration(duration);
```
[the remark this answers is not in the document]
Comment (dmoree) [reply packages/graphql/tests/integration/types/duration.int.test.ts:58]: This will be somewhat difficult to test in that case because the driver doesn't allow passing a string as a duration. Since it only takes days months seconds and nanoseconds, the string stored in the database that represents the duration is not necessarily the same as the one being defined in the test. I would have to recreate that function in this test otherwise.

This is why there is an explicit `Duration.test.ts` file that tests various strings to make sure `parseDuration` is working correctly. It just happened that when using floating point numbers from `faker` it was randomly exposing an edge case not covered by that test.

File: packages/graphql/tests/integration/types/duration.int.test.ts [hunk 2]
```diff
@@ -52,12 +51,10 @@ describe("Duration", () => {
             });
 
             const id = generate({ readable: false });
-            const years = faker.random.number({ min: -6, max: 6 });
-            const months = faker.random.number({ min: -10, max: 10 });
-            const days = faker.random.number({ min: -50, max: 50 });
-            const minutes = faker.random.float();
+            const hours = 2;
```
[the remark this answers is not in the document]
Comment (dmoree) [reply packages/graphql/tests/integration/types/duration.int.test.ts:57]: Added back in https://github.com/neo4j/graphql/pull/774/commits/2a2ee4932429becbac7b40a4c35dc3ed15403f6b

**Review by angrykoala — APPROVED** [review]

File: packages/graphql/tests/integration/types/duration.int.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (angrykoala) [reply packages/graphql/tests/integration/types/duration.int.test.ts:61]: Fair enough, this proposal was outside the scope of the PR anyway, so happy to approve as is

File: packages/graphql/tests/integration/types/duration.int.test.ts [hunk 3]
```diff
@@ -94,7 +94,7 @@ describe("Duration", () => {
                     { id }
                 );
 
-                const neo4jMovie: { id: string; duration: any } = neo4jResult.records[0].toObject().movie;
+                const neo4jMovie: { id: string; duration: object } = neo4jResult.records[0].toObject().movie;
```
Comment (angrykoala) [inline packages/graphql/tests/integration/types/duration.int.test.ts:97]: :clap:
